helm-values/traction/values-sandbox.yaml

```
@@ -116,10 +116,13 @@ ingress:
   annotations:
     route.openshift.io/termination: edge
 postgresql:
-  resources:
-    limits:
-      cpu: 400m
-      memory: 1600Mi
-    requests:
-      cpu: 200m
-      memory: 820Mi
+  primary:
+    persistence:
+      size: 5Gi
+    resources:
+      limits:
+        cpu: 400m
+        memory: 1600Mi
```

```suggestion
        cpu: 2
        memory: 4000Mi
```

During load testing the Postgres instance was getting heavily throttled (>80%), so I updated the limits.  Let's make those a permanent change.